pkg/virt-controller/network/render_test.go

```diff
@@ -0,0 +1,64 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2024 Red Hat, Inc.
+ *
+ */
+
+package network
```

Comment:
It is not testable if we use a test package?

Reply:
It is doable, but some non-exported methods with have to be exported.

Reply:
Ok, lets consider this for a follow up then.

Reply:
> It is doable, but some non-exported methods with have to be exported.

Or modify the tests, so they validate only exported functions. ;-)